packages/application-shell/package.json

```diff
@@ -82,7 +82,7 @@
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.26",
     "omit-empty-es": "1.0.3",
-    "perfume.js": "2.1.2",
+    "perfume.js": "3.0.4",
```

Comment:
Nothing breaking for us here since the last update to v2 last week. It now supporst navigation timing which could be cool.

<img width="813" alt="CleanShot 2019-10-22 at 09 45 40@2x" src="https://user-images.githubusercontent.com/1877073/67266737-365d8c80-f4b1-11e9-8cb1-74884c384eed.png">